src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  );
+}
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
interesting approach, but you can set the default variables as you pass the props with `a = 0`